### ISSUE TITLE
Reduce spacing between paragraphs

### ISSUE THREAD

**Issue by mattheu** [humanmade/hm-pattern-library#143]
Currently we have `margin: 1.66667rem 0` on paragraphs. This is consistent with most of the margins across the pattern library. However it is quite a big gap when reading long blocks of text. It is particularly noticeable on H2 where we have more shorter paragraphs.

I propsose we drop it by half.

Current:

![image](https://user-images.githubusercontent.com/494927/31887529-9dd9a204-b7f0-11e7-9271-6a4a8ab0ddd7.png)

New

![image](https://user-images.githubusercontent.com/494927/31887540-adcd8ed2-b7f0-11e7-8ba9-83159e194939.png)

**Comment by grarighe** [comment]
Good point @mattheu. By looking at it I'd suggest a 1 rem. What do you think?
